bin/check-influxdb-query.rb

```diff
@@ -114,7 +114,7 @@ class CheckInfluxdbQuery < Sensu::Plugin::Check::CLI
          short: '-m MODE',
          long: '--mode MODE',
          default: 'first',
-         description: 'How the results are being checked (one of "first", "last", "max", "min", "avg") when the query returns more than one value'
+         description: 'How the results are being checked (one of "first", "last", "max", "min", "sum", "avg") when the query returns more than one value'
```

Comment:
Makes sense to me, we should probably configure the list of acceptable inputs so that if someone typo's something or tries a function not yet supported they will get better and quicker feedback. Also rather than relying on implicit logic driven case statement it's probably just better to default to `first`
```suggestion
         description: 'How the results are being checked (one of "first", "last", "max", "min", "sum", "avg") when the query returns more than one value',
         in: %w(first last max min sum avg average),
         default: 'first'
```

Reply:
Sounds good. Applied the suggestion :)